src/_guides/openapi/openapi-format-overlays.md

```diff
@@ -0,0 +1,96 @@
+
+## What is OpenAPI Format
+
+[OpenAPI Format](https://github.com/thim81/openapi-format) started life as a NPM command-line tool, with the goal of helping to tidy up OpenAPI documents. It can remove unused components, sort various components and parameters alphabetically, upgrade documents from OpenAPI v3.0 to v3.1, and all kinds of other handy utilities to keep OpenAPI documents neat and tidy.
+
+Recently OpenAPI Format got the ability to apply overlays to OpenAPI documents, but they didn’t stop with a command-line interface for this.
+
+## Introducing OpenAPI-Format Playground
+
+The maintainers of OpenAPI-Format built a Playground, a web interface to help folks work with overlays in a visual way. 
+
+Overlays are essentially a series of “actions” which can be applied to a document, in order to create a new document with the resulting changes. Actions need a target defined as a [JSONPath](/guides/openapi/jsonpath/) to point to a particular part of an OpenAPI document, and an action type of of “update” or “remove”. These actions can be used in combination to add new content, replace content, or filter out bits that aren’t wanted.
+
+Figuring out how to do all of this just from staring into an empty text editor can be a bit rough, so let’s try using the playground to guide us on our way.
+
+## Creating Overlays with OpenAPI-Format Playground
+
+### Step 1.) Add an OpenAPI document
+
+Copy in some OpenAPI into the box on the left. If you don’t have any handy why not use the [Train Travel API](https://github.com/bump-sh-examples/train-travel-api) by Bump.sh.
+
+![](new-openapi-in/images/guides/openapi-format-overlays/new-openapi-in-playground.png)
+
+### Step 2.) Open the Overlay section
+
+Click the “*OpenAPI Overlay”* button and it will go to a different section. In here you can give your Overlay a name, something that describes what we are trying to do. 
+
+For example: “Add descriptions to tags” is pretty clear about what is going to be achieved by the actions of this overlay. The version number can be whatever. 
+
+![](/images/guides/openapi-format-overlays/overlay-name.png)
+
+### Step 3.) Play with JSONPath
+
+JSONPath is hard and scary at first, but much like RegEx it can be mastered, especially with the help of a handy tool like this. 
+
+To start with JSONPath you use the `$` as the root, then `.` to go to a property. The OpenAPI structure can be navigated like this, so something like `$.info` `$.paths` `$.tags` or `$.components` is a good start.
+
+When you type in a JSONPath in the Target field, the Target Preview will update, so you can be certain you are working on the right bit of the OpenAPI document.
+
+![](/images/guides/openapi-format-overlays/overlay-target.png)
+
+To update a specific tag, JSONPath supports filters, which use a `?(condition)` syntax. Here I am looking through an array of objects for the one which has a name property equal to “Stations”:
```

Comment:
```suggestion
To update a specific tag, JSONPath supports filters, which use a `?(condition)` syntax. Here I am looking through an array of objects for the one which has a name property equal to “Stations”: `$.tags[?(@.name == "Stations")]`

```
As it's a bit cumbersome at first to play with the syntax, let's make this example copy/pastable